src/hooks/useIconColor.ts

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react'
+import { getColors } from 'react-native-image-colors'
+
+import { useState } from '../types/reactHooks'
+import { EdgeAsset } from '../types/types'
+import { getCurrencyIconUris } from '../util/CdnUris'
+
+export const useIconColor = (edgeAsset: EdgeAsset): string | undefined => {
```

Comment:
Why aren't we using this this `TransactionListScene` for consistency?

Reply:
My suggestion was to gut the icon grab out of the `CyptoIconUi4` in favor of this hook. But, no worries if not.